<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CyberVault: Password Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/logo/icon.ico</normaloff>
    <normalon>:/logo/icon.ico</normalon>
    <disabledoff>:/logo/icon.ico</disabledoff>
    <disabledon>:/logo/icon.ico</disabledon>
    <activeoff>:/logo/icon.ico</activeoff>
    <activeon>:/logo/icon.ico</activeon>
    <selectedoff>:/logo/icon.ico</selectedoff>
    <selectedon>:/logo/icon.ico</selectedon>:/logo/icon.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(141, 145, 141);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>920</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>CyberVault: Password Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>920</width>
      <height>115</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Your one stop shop for RSA based password management. This system will allow you to generate an RSA key pair to keep all your password information safe. The CyberVault can also be used with an Authenticator App to secure your vault with MFA.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="new_account">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>395</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 208, 162);</string>
    </property>
    <property name="text">
     <string>New Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_cybervault">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>395</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 208, 162);</string>
    </property>
    <property name="text">
     <string>Import CyberVault</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_to_account">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>285</y>
      <width>835</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 208, 162);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <zorder>Title</zorder>
   <zorder>info</zorder>
   <zorder>login_to_account</zorder>
   <zorder>import_cybervault</zorder>
   <zorder>new_account</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionOpen_Vault"/>
    <addaction name="actionExport_Vault"/>
    <addaction name="pass_gen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAccount"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="pass_gen">
   <property name="text">
    <string>Generate Password</string>
   </property>
  </action>
  <action name="exit_app">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_Vault">
   <property name="text">
    <string>Import Vault</string>
   </property>
  </action>
  <action name="actionExport_Vault">
   <property name="text">
    <string>Export Vault</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>